pygmt/datasets/load_remote_dataset.py

```diff
@@ -226,6 +226,29 @@ class GMTRemoteDataset(NamedTuple):
             "03m": Resolution("03m", registrations=["gridline"], tiled=True),
         },
     ),
+    "moon_relief": GMTRemoteDataset(
+        title="Moon relief",
+        name="moon relief",
+        long_name="Moon relief",
```

Comment:
Please see https://www.generic-mapping-tools.org/remote-datasets/moon-relief.html.

```suggestion
        title="Moon relief",
        name="moon relief",
        long_name="USGS Moon (LOLA) relief ",
```